### PR TITLE
Feature/refresh eq surveys

### PR DESCRIPTION
**Pull request by gemmai95**

### CODE REVIEW

File: transform/surveys/023.0102.json [hunk 1]
```diff
@@ -27,19 +99,39 @@
       ]
     },
     {
-      "title": "4. Retail Turnover",
+      "title": "3. Commodoties - Retail Turnover",
       "meta": "(£)",
       "questions": [
         {
+          "text": "Food",
```
Comment (warrenbailey) [inline transform/surveys/023.0102.json]: Is this correct? Are we adding questions to this form type?

File: transform/surveys/023.0203.json [hunk 2]
```diff
@@ -5,15 +5,87 @@
...
+            "type": "checkbox",
+            "options": [
+              "Other"
+            ]
+          }
+        ]
     },
     {
-      "title": "1. Period",
+      "title": "2. Period",
```
Comment (warrenbailey) [inline transform/surveys/023.0203.json]: Why have the title numbers changed?

Comment (gemmai95) [inline transform/surveys/023.0203.json]: Does the title numbers not have to match that in the refreshed survey?

File: transform/surveys/023.0102.json (hunk 1 above)
Comment (gemmai95) [inline transform/surveys/023.0102.json]: This matches what was added for the survey that was already refreshed. Does this mean new questions that have been added shouldn't be included?

Comment (warrenbailey) [inline transform/surveys/023.0102.json]: Different form types have different questions, so we should only change the questions that are currently there and not add new ones 👍

File: transform/surveys/023.0203.json (hunk 2 above)
Comment (warrenbailey) [inline transform/surveys/023.0203.json]: It should match the EQ ones, but I think they can probably stay the same. Might be worth checking on the EQ dev page

File: transform/surveys/023.0102.json (hunk 1 above)
Comment (gemmai95) [inline transform/surveys/023.0102.json]: I checked with the EQ dev page and all the extra questions should be gone I think

File: transform/surveys/023.0203.json (hunk 2 above)
Comment (gemmai95) [inline transform/surveys/023.0203.json]: Where exactly do you get the question title on the dev page surveys as they don't seem to be there when I looked

Comment (warrenbailey) [inline transform/surveys/023.0203.json]: Good point, there isn't any in EQ 

Then definitely don't change them :)

File: transform/surveys/023.0205.json [hunk 3]
```diff
@@ -28,7 +100,7 @@
       ]
     },
     {
-      "title": "2. Commodoties - Retail Turnover",
+      "title": "3. Commodoties - Retail Turnover",
```
Comment (warrenbailey) [inline transform/surveys/023.0205.json]: Lets put these back to the original numbers

File: transform/surveys/023.0102.json [hunk 4]
```diff
@@ -27,19 +99,9 @@
       ]
     },
     {
-      "title": "4. Retail Turnover",
-      "meta": "(£)",
-      "questions": [
-        {
-          "text": "Total Retail Sales",
-          "question_id": "20"
-        }
```
Comment (warrenbailey) [inline transform/surveys/023.0102.json]: I don't think this question should be removed?

File: transform/surveys/023.0112.json [hunk 5]
```diff
@@ -4,16 +4,84 @@
...
   "question_groups": [
     {
-      "title": "Comments",
-      "questions": [{
-        "text": "Please indicate the reasons for any changes in the business's total retail turnover",
-        "question_id": "146",
-        "type": "contains"
-      }, {
-        "text": "Explain any significant changes in the number of employees",
-        "question_id": "147",
```
Comment (warrenbailey) [inline transform/surveys/023.0112.json]: We still need this one (147) too

File: transform/surveys/023.0203.json [hunk 6]
```diff
@@ -28,7 +100,7 @@
       ]
     },
     {
-      "title": "2. Commodoties - Retail Turnover",
+      "title": "2.Commodoties - Retail Turnover",
```
Comment (warrenbailey) [inline transform/surveys/023.0203.json]: Missing space

File: tests/pck/023.0102.nobatch [hunk 7]
```diff
@@ -3,6 +3,5 @@ RSI5B:12345678901A:1604
 0001 00000000002
 0011 00000010416
 0012 00000311016
-0020 00001800000
```
Comment (warrenbailey) [inline tests/pck/023.0102.nobatch]: Again this is needed

File: tests/pck/023.0112.employee.comments.json [hunk 8]
```diff
@@ -21,7 +21,6 @@
     "54": "24",
     "50": "205",
     "21": "60000",
-    "146": "A comment for 0112",
-    "147": "An employee comment for 0112"
```
Comment (warrenbailey) [inline tests/pck/023.0112.employee.comments.json]: Likewise we need question 147

File: tests/pck/023.0102.pck [hunk 9]
```diff
@@ -4,6 +4,5 @@ RSI5B:12345678901A:1604
 0001 00000000002
 0011 00000010416
 0012 00000311016
-0020 00001800000
```
Comment (warrenbailey) [inline tests/pck/023.0102.pck]: Test data shouldn't change, we need these to be made. The new schemas must be slightly wrong.

File: tests/pck/023.0102.json [hunk 10]
```diff
@@ -16,7 +16,6 @@
   "data": {
     "11": "1/4/2016",
     "12": "31/10/2016",
-    "20": "1800000",
```
Comment (warrenbailey) [inline tests/pck/023.0102.json]: We need this too

File: transform/surveys/023.0112.json (hunk 5 above)
Comment (gemmai95) [inline transform/surveys/023.0112.json]: 147 has been added back in at the bottom of question 146

**Comment by gemmai95** [comment]
[(3) EQ Survey Refresh](https://trello.com/c/X9JTGGuI/162-3-eq-survey-refresh)

File: transform/surveys/023.0112.json (hunk 5 above)
Comment (warrenbailey) [inline transform/surveys/023.0112.json]: Missed that sorry

File: transform/surveys/023.0112.json [hunk 11]
```diff
@@ -31,10 +104,30 @@
       ]
     },
     {
-      "title": "3. Retail Turnover",
+      "title": "3. Commodoties - Retail Turnover",
       "meta": "(£)",
       "questions": [
         {
+          "text": "Food",
```
Comment (warrenbailey) [inline transform/surveys/023.0112.json]: I don't think we need to add these questions. Looking at form type 0112 on https://preprod-surveys-launch.eq.ons.digital/ it only has Total Retail Sales.

So you can remove questions 22, 23, 24, 25 and 26

**Pull request merged**